CondeNast/launch-vehicle-fbm
# Add Readme to Sphinx documentation

## Why are we doing this?

If you look at the current documentation site: https://launch-vehicle-fbm.readthedocs.io/en/latest/ You'll notice the homepage is blank. This hacks a way to get the README to take that empty space.

## Did you document your work?

The Makefile has all the commands. It's not obvious that you need to build the docs if you update the README right now, but I'm hoping that future tooling will make this easier. It's sort of acceptable to let `index_readme.rst` to get out of sync right now because the tooling ain't great.

## How can someone test these changes?

Steps to manually verify the change:

1. `cd docs`
2. `make install`
3. `make build` -- index_readme.rst should get built.
4. `make watch`
5. browse to http://localhost:8000 -- you should see the README presented

## What possible risks or adverse effects are there?

* no code changes, this just makes the docs better
* we're storing generated docs in source control. That means if the README change, there's double the lines changed

## What are the follow-up tasks?

* none

It's currently not possible, but it'd be nice if the ReadTheDocs build process could do the build. That way, we could at least put `index_readme.rst` in `.gitignore`

## Are there any known issues?

none

## Did the test coverage decrease?

_(Optional)_ Test coverage: increasing test coverage is important to us. If you let test
coverage slip, you have to justify it.
